perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -1512,7 +1515,7 @@ sub waste_reconcile_direct_debits {
             # processed reports so we can warn on those we are missing.
             $rs = $rs->search({ category => 'Garden Subscription' });
             while ( my $cur = $rs->next ) {
-
+                next unless $cur->get_extra_field_value('payment_method') && $cur->get_extra_field_value('payment_method') eq 'direct_debit';
```

Comment:
Could these three repeated lines be moved into a method, like `$cur->is_direct_debit` or similar?